internal/pkg/exporter/stackdriver/stackdriver.go

```diff
@@ -0,0 +1,48 @@
+package stackdriver
+
+import (
+	"context"
+	"cse/internal/pkg/exporter"
+	"fmt"
+	"log"
+
+	"contrib.go.opencensus.io/exporter/stackdriver"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+// StackDriver exporter implementation
+type StackDriver struct {
+}
+
+// Record new stat
+func (sd *StackDriver) Record(ctx context.Context, stat *stats.Float64Measure) {
+
+}
+
+// Register new stat
+func (sd *StackDriver) Register(lineCountView *view.View) (exporter.Exporter, error) {
+	fmt.Println("Registering for StackDriver")
+	if err := view.Register(lineCountView); err != nil {
+		log.Fatalf("Failed to register the views: %v", err)
+	}
+
+	// TODO: Define each exporter in a separate file/package
+	exporter, err := stackdriver.NewExporter(stackdriver.Options{})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer exporter.Flush()
+	if err := exporter.StartMetricsExporter(); err != nil {
+		log.Fatalf("Error starting metric exporter: %v", err)
+	}
+	defer exporter.StopMetricsExporter()
```

Comment:
Because you are defering here, once the function is over (reaches the return) the exporter is stopped.

Maybe as a suggestion of refactoring here is that we could have methods like Flush and Stop into the interface, and when the exporter does not have this (like Prometheus) simple return.